src/app/content/components/Content.tsx

```diff
@@ -47,18 +47,6 @@ const Background = styled.div`
   }
 `;
 
-// tslint:disable-next-line:variable-name
-const ContentNotifications = styled(Notifications)`
```

Comment:
we need this here because the top offset is different in the `content` module than outside of it (because of the book banner and toolbar)